Good-Bids/goodbids
# [#507] Setting up basic site setup page

# Summary

This adds a basic site setup page in the WP admin. The page content will be added in another ticket. 

## Issues

* #507

## Testing Instructions

1. Go to the site admin and make sure you see the new "Site Setup" at the top of the navigation.

## Screenshots

<img width="779" alt="Screenshot 2024-02-27 at 1 30 02 PM" src="https://github.com/Good-Bids/goodbids/assets/91974372/5b238615-05c6-43a1-ae8e-b0a1dc3b4fb9">
